Makefile

```diff
@@ -39,7 +39,7 @@ wait-healthy: ## Wait for the containers to be healthy
 		echo "No containers running"; \
 		exit 1; \
 	fi; \
-	for service in $${services}; do \
+	for service in $${services[@]}; do \
```

Comment:
bugfix seems correct from quick experimentation, arcane Make syntax

Reply:
Never seen it before, but https://stackabuse.com/array-loops-in-bash/#loopingoverarrays suggests it's Bash?